example/mapboxExample.js

```diff
@@ -0,0 +1,176 @@
+import * as THREE from 'three';
+import CameraSync, {
+	projectToWorld,
+	projectedUnitsPerMeter,
+} from './mapboxExampleCamera.js';
+import { DebugTilesRenderer as TilesRenderer } from '../src/index.js';
+import { GUI } from 'three/examples/jsm/libs/lil-gui.module.min.js';
+
+const params = {
+
+	'accessToken': 'pk.eyJ1IjoieGlheGlhbmdmbmVnIiwiYSI6ImNscTRzc245NTA5c3cya3BhdXA4MDY0NWQifQ._9T_J9z7Hvcd00nQL3UuSw',
+	'reload': reload,
+
+};
+
+// GUI
+const gui = new GUI();
+gui.width = 300;
+
+const mapboxglOptions = gui.addFolder( 'mapboxgl' );
+mapboxglOptions.add( params, 'accessToken' );
+mapboxglOptions.add( params, 'reload' );
+mapboxglOptions.open();
+
+function rotationBetweenDirections( dir1, dir2 ) {
+
+	const rotation = new THREE.Quaternion();
+	const a = new THREE.Vector3().crossVectors( dir1, dir2 );
+	rotation.x = a.x;
+	rotation.y = a.y;
+	rotation.z = a.z;
+	rotation.w = 1 + dir1.clone().dot( dir2 );
+	rotation.normalize();
+
+	return rotation;
+
+}
+
+let map;
+const origin = [ 116, 35 ];
```

Comment:
What are the units fore these values? Are these lat / lon values?

Reply:
Yes, it's longitude and latitude